node/api/examples/channels/legacy/continuous/init_per_group.md

```diff
@@ -0,0 +1,276 @@
+  "channel_id": null,
+  "payload": [
+    {
+      "account": "ak_6LRvzWhqyZL2c4Z2Vzocr5SsfcaR3T515Wm34m62u2xsaGMqn",
+      "balance": 69999999999999
+    },
+    {
+      "account": "ak_2FqziTRw9osxE289AHiNo4s2GN2c8i52RpLaiwTjqCbY5YREq9",
+      "balance": 40000000000001
+    }
+  ],
+  "tag": "balances",
+  "version": 1
+}
+```
+
+#### responder <--- node (2019-03-13 11:13:48.476)
+```javascript
+{
+  "action": "info",
+  "channel_id": null,
+  "payload": {
+    "event": "own_funding_locked"
+  },
+  "version": 1
+}
+```
+
+#### initiator <--- node (2019-03-13 11:13:48.480)
+```javascript
+{
+  "action": "info",
+  "channel_id": null,
+  "payload": {
+    "event": "own_funding_locked"
+  },
+  "version": 1
+}
+```
+
+#### initiator <--- node (2019-03-13 11:13:48.480)
+```javascript
+{
+  "action": "info",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "event": "funding_locked"
+  },
+  "version": 1
+}
+```
+
+#### responder <--- node (2019-03-13 11:13:48.486)
+```javascript
+{
+  "action": "info",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "event": "funding_locked"
+  },
+  "version": 1
+}
+```
+
+#### initiator <--- node (2019-03-13 11:13:48.501)
+```javascript
+{
+  "action": "info",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "event": "open"
+  },
+  "version": 1
+}
+```
+
+#### initiator <--- node (2019-03-13 11:13:48.503)
+```javascript
+{
+  "action": "update",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "state": "tx_+QENCwH4hLhAmZbbVf4/NfmhVbnLEUzxP1zb/m/r1mHsBzQRIB5Ku2bGrKb7o6MvAcjX6PKSrdt4hsciL6H0xFnDRp5V3cJLAbhAoI+fvXfrhJ4kjSz+lgij4CS9ANvHmsDrvbkJeODGA6PHCNxnors/VotUYUZeLuM2F8mdSBBbFFTBSjK/yZQjBriD+IEyAaEBDB0foygfU35mpXAMOB5T3y5Uz+HxiwugHh8Qy/jI3nqGP6olImAAoQGlZyRxc6JXyoCZ1k5JANokwX6uj8/EZxGEGyc8x2MXK4YkYTnKgAACCgCGEjCc5UAAwKAVUCLa13l4tTTVS6xTUZMu5qobq+gq1dyIaegDWACVUga6kKf1"
+  },
+  "version": 1
+}
+```
+
+#### responder <--- node (2019-03-13 11:13:48.507)
+```javascript
+{
+  "action": "info",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "event": "open"
+  },
+  "version": 1
+}
+```
+
+#### responder <--- node (2019-03-13 11:13:48.511)
+```javascript
+{
+  "action": "update",
+  "channel_id": "ch_2gouzi4HtoExEMDucQ96sT4da6wg73cvJuD7Su3Uwqaaq3bFtT",
+  "payload": {
+    "state": "tx_+QENCwH4hLhAmZbbVf4/NfmhVbnLEUzxP1zb/m/r1mHsBzQRIB5Ku2bGrKb7o6MvAcjX6PKSrdt4hsciL6H0xFnDRp5V3cJLAbhAoI+fvXfrhJ4kjSz+lgij4CS9ANvHmsDrvbkJeODGA6PHCNxnors/VotUYUZeLuM2F8mdSBBbFFTBSjK/yZQjBriD+IEyAaEBDB0foygfU35mpXAMOB5T3y5Uz+HxiwugHh8Qy/jI3nqGP6olImAAoQGlZyRxc6JXyoCZ1k5JANokwX6uj8/EZxGEGyc8x2MXK4YkYTnKgAACCgCGEjCc5UAAwKAVUCLa13l4tTTVS6xTUZMu5qobq+gq1dyIaegDWACVUga6kKf1"
+  },
+  "version": 1
+}
+```
+
+#### responder: (2019-03-13 11:13:48.972)
+> Funding has been confirmed locally on-chain
```

Comment:
This comments shouldn't probably all be at the end. I think some write caching happened probably. (output wasn't flushed correctly and all messages were printed at the end.) This also applies to other transcripts, but weirdly to not all of them.

Reply:
It was a race condition, addressed in [80293cd](https://github.com/aeternity/aeternity/pull/2235/commits/80293cd7f62052d6c5ef4ea7de5eee89c5c23f00)